rma/models/rma.py

```diff
@@ -788,8 +788,6 @@ def action_preview(self):
     def action_view_receipt(self):
         """Invoked when 'Receipt' smart button in rma form view is clicked."""
         self.ensure_one()
```

Comment:
Has this patch being lost?

Reply:
I don't think so, seems to work just fine. Example using rma_sale, which adds a smart button on a SO, clicking through the smart button on the SO, the smart buttons on the RMA work correctly with this change.